templates/pages/home.jsx

```diff
@@ -1,12 +1,80 @@
-var React = require('react');
+var React = require('react'),
+    BadgeList = require('../components/badgelist.jsx')
 
 var Home = React.createClass({
   componentDidMount: function() {
-    document.title = "PaperBadger";
+    document.title = "Contributorship Badges";
   },
   render: function() {
     return (
-      <div>Welcome to PaperBadger</div>
+      <div>
+        <div>
+        <h1>Contributorship Badges</h1>
+        <p>Exploring the use of digital badges for crediting contributors to scholarly papers for their work. <a href="/about">More about contributorship badges.</a></p>
+        </div>
+        <h2>Explore Badges</h2>
+        <ul>
+          <li><a href="/issue">Issue a badge</a></li>
+          <li><a href="/users/0000-0003-4959-3049/badges">Browse issued badges for ORCID 0000-0003-4959-3049</a> (JSON endpoint)</li>
```

Comment:
Might be better using <Link/> provided by react router for all internal pages here?...